hack/prebuild/init.sh

```diff
@@ -0,0 +1,19 @@
+#/bin/bash
+
+yum install -y -d1 libvirt libvirt-daemon-kvm
+usermod -aG libvirt root
```

Comment:
Not sure I've looked to see how this script is used (or when) but do you not need `newgrp libvirt` here having modified the groups?

Reply:
It is used to install libvirt on packet host the easiest way.